deploy/environments/libraries.yaml

```diff
@@ -41,7 +41,10 @@ spack:
     - morpho-kit
     - mvdtool+mpi
     - neuron+mpi~debug%intel
-    - neuron+coreneuron~legacy-unit%nvhpc ^coreneuron+gpu~legacy-unit~report ^py-cython%gcc ^py-numpy%gcc
+    # This is similar to what is built in the GitLab CI for CoreNEURON; the
+    # idea is to reduce the number of dependencies that the CI has to re-build
+    # on every run. 
```

Comment:
I would remove the comment lines from this yaml file. Mostly us (neuron/coreneuron devs) will be looking at neuron related specs here.